appinventor/components-ios/src/Sprite.swift

```diff
@@ -403,7 +403,53 @@ open class Sprite: ViewComponent, UIGestureRecognizerDelegate {
     updateY(y: y)
     registerChanges()
   }
+
```

Comment:
These changes are from your other PR and are not relevant to this PR. We should clean up the branch before merging.